src/lib/controllers/tokens-edit-controller.js

```diff
@@ -0,0 +1,76 @@
+import L10nSingleton from '@/lib/l10n/l10n-singleton.js'
+import NotificationSingleton from '@/lib/notifications/notification-singleton'
+
+export default class TokensEditController {
+  /**
+   * @param {Vuex Store} store
+   */
+  constructor (store) {
+    this.store = store
+  }
+
+  uploadAlignment (alignment) {

```

Comment:
upload is a confusing name here, since we have an upload function.  Maybe just 'loadAlignment' ?

Reply:
oh, youa re right - will update it

Reply:
done